pkg/kubelet/config/file_linux_test.go

```diff
@@ -51,7 +51,7 @@ func TestExtractFromNonExistentFile(t *testing.T) {
 }
 
 func TestUpdateOnNonExistentFile(t *testing.T) {
-	ch := make(chan interface{})
+	ch := make(chan interface{}, 1)
```

Comment:
what the point to make it unblocking channel when we have select anyway?

Reply:
@cynepco3hahue There is a child goroutine created in NewSourceFile ([line 95 in file.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/config/file.go#L95)). The child goroutine sends a message to ch ([line 129 in file.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/config/file.go#L129)). If the select chooses the timeout case, the child goroutine is leaked.